.goreleaser.yaml

```diff
@@ -29,6 +29,16 @@ builds:
     goarch: *goarch
```

Comment:
BTW If we merge this here, then someone following the blog post will get the gh plugin out of the box, right? so maybe we should change the name in the blogpost to something different?